docs/references/plugin-compatibility.md

```diff
@@ -0,0 +1,134 @@
+|tcp-log                         |  :white_check_mark:              |  :white_check_mark:                        |
+|udp-log                         |  :white_check_mark:              |  :white_check_mark:                        |
+|zipkin                          |  :white_check_mark:              |  :white_check_mark:                        |
+|application-registration        |  :white_check_mark:              |  :x:<sup>1</sup>                           |
+|canary release                  |  :white_check_mark:              |  :white_check_mark:                        |
+|collector                       |  :white_check_mark:              |  :white_check_mark:                        |
+|degraphql                       |  :white_check_mark:              |  :white_check_mark:                        |
+|exit-transformer                |  :white_check_mark:              |  :x:                                       |
+|forward-proxy                   |  :white_check_mark:              |  :white_check_mark:                        |
+|graphql-proxy-cache-advanced    |  :white_check_mark:              |  :white_check_mark:                        |
+|graphql-rate-limiting-advanced  |  :white_check_mark:              |  :white_check_mark:                        |
+|jwt-signer                      |  :white_check_mark:              |  :white_check_mark:                        |
+|kafka-log                       |  :white_check_mark:              |  :white_check_mark:                        |
+|kafka-upstream                  |  :white_check_mark:              |  :white_check_mark:                        |
+|key-auth-enc                    |  :white_check_mark:              |  :x:                                       |
+|ldap-auth-advanced              |  :white_check_mark:              |  :white_check_mark:                        |
+|mtls-auth                       |  :white_check_mark:              |  :white_check_mark:                        |
+|oauth2-introspection            |  :white_check_mark:              |  :white_check_mark:                        |
+|openid-connect                  |  :white_check_mark:              |  :white_check_mark:                        |
+|proxy-cache-advanced            |  :white_check_mark:              |  :white_check_mark:                        |
+|rate-limiting-advanced          |  :white_check_mark:              |  :white_check_mark:                        |
+|request-transformer-advanced    |  :white_check_mark:              |  :x:<sup>2</sup>                           |
+|request-validator               |  :white_check_mark:              |  :white_check_mark:                        |
+|response-transformer-advanced   |  :white_check_mark:              |  :white_check_mark:                        |
+|route-transformer-advanced      |  :white_check_mark:              |  :x:                                       |
```

Comment:
Let's add an explanation for this as well.

Here is what I found from internal slack:
> Route transformer advanced is not available because those features are available in other forms. You can use route paths and service path or use request-transformer to implement those changes.

Reply:
I don't believe that's the case for this one--documentation is pretty scant and I didn't dig into the code, but it looks like it uses some sort of templating system to apply the changes, and is indeed different from standard route/service configuration (or if nothing else, it does the same thing, but can be applied on a per-consumer basis).

It doesn't look like there was any clear reason to omit it (no DAOs and no other obvious conflict I can think of)--it seems like exit-transformer where I think the most likely explanation is that it was just added to Enterprise after we created the kong-enterprise-k8s kong-distributions fork, and then never found its way into build.yaml. Should we just add it there?